lint-configs/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@discourse/lint-configs",
+  "version": "1.0.0",
+  "description": "Shareable lint configs for Discourse core, plugins, and themes",
+  "author": "Discourse",
+  "license": "MIT",
+  "keywords": [
+    "discourse"
+  ],
+  "repository": "https://github.com/discourse/eslint-config-discourse",
+  "homepage": "https://github.com/discourse/eslint-config-discourse#readme",
+  "type": "module",
+  "exports": {
+    "./eslint": {
+      "require": "./eslintrc.cjs"
+    },
+    "./eslint-theme": {
+      "require": "./eslint-theme.cjs"
+    },
+    "./prettier": {
+      "require": "./.prettierrc.cjs"
+    },
+    "./template-lint": {
+      "require": "./template-lint.config.cjs"
+    }
+  },
+  "scripts": {
+    "lint": "eslint --no-error-on-unmatched-pattern **/*.cjs **/*.js && pnpm prettier --check .",
+    "test": "cd ../test && node test.js"
+  },
+  "dependencies": {
+    "@babel/core": "^7.23.2",
+    "@babel/eslint-parser": "^7.22.15",
+    "@babel/plugin-proposal-decorators": "^7.23.2",
+    "ember-template-lint": "^5.11.2",
+    "eslint": "^8.51.0",
+    "eslint-plugin-decorator-position": "^5.0.2",
+    "eslint-plugin-discourse-ember": "latest",
+    "eslint-plugin-ember": "^11.11.1",
+    "eslint-plugin-simple-import-sort": "^10.0.0",
+    "eslint-plugin-sort-class-members": "^1.19.0",
```

Comment:
Do these need to be peerDependencies as well? (right now in core, we include them as top-level dependencies... but maybe its no longer required?)

Reply:
In my testing - no. We only need those packages which bin's we're using: eslint, prettier, and ember-template-lint.

And even with those, it's not required when using yarn and will only  be needed with pnpm.

And even then, pnpm has built-in compatibility for eslint and prettier so we could only add ember-template-lint, but for correctness sake I added all three. 😛